pandas/lib.pyx

```diff
@@ -1055,6 +1055,20 @@ def string_array_replace_from_nan_rep(ndarray[object, ndim=1] arr, object nan_re
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
+def object_array_decode_bytes(ndarray[object, ndim=1] arr, object encoding):
+    """Decode any instances of bytes to str in arr using the given encoding."""
```

Comment:
again this is duplicating code, you can simply use ``core.strings.str_decode``  (but needs to be in ``to_native_types``)